src/TestSetExtensions.jl

```diff
@@ -57,6 +57,17 @@ function ExtendedTestSet(desc; wrap=DefaultTestSet)
     ExtendedTestSet{wrap}(desc)
 end
 
+function isVector(e)
+    if e.head === :vect
+        return true
+    end
+    #Float32 or Int32 arrays get here as Ref's to Vector
+    if e.head === :ref && isa(e.args, Vector)
+        return true
+    end
+    return false
```

Comment:
Since `e::Expr`, we have `isa(e.args, Vector)` to always be true. So the following suffice
```suggestion
    return e.head === :ref
```